README.md

```diff
@@ -116,34 +116,113 @@ This allow to **configure** the layouts **on the go** while using it.
+5. Note that a few Material-Shell keybindings duplicate some Gnome keybindings e.g. `Super+s, Super+PgDown`, `Super+Home`, `Super+End` etc..
+
 ## Hotkeys
-Some hotkeys might already be used by GNOME Shell - please check your keybindings first.
-#### Desktop navigation
-* `Super+W` Navigate to the upper workspace/category.
-* `Super+S` Navigate to the lower workspace/category.
-* `Super+A` Focus the window at the left of the current window.
-* `Super+D` Focus the window at the right of the current window.
-* `Super+1`, `Super+2` ... `Super+0` Navigate to specific workspace
-
-#### Window manipulation
-* `Super+Q` Kill the current window focused.
+Material-Shell is often used in conjunction with Gnome keyboard-shortcuts, Firefox shortcuts, etc. 
+Note that some hotkeys might already be used by other applications - please check your keybindings first.
+
+#### Choosing tiling layout
+* `Super+Space` Cycle through the tiling options in the current workspace.
+
+#### Open and close a window
+* `Super+x` Open the App Launcher within a new window.
+* `Super+q` Kill the current focused window.
+
+#### Moving the focus aka Navigation
+... window panes 
+* `Super+d` Focus on the next window pane, left to right.
+* `Super+a` Focus on the previous window pane, right to left.
+
+...workspaces 1 through 10
+* `Super+1`, `Super+2` ... `Super+0` Navigate to specific workspace 1 - 10
+* `Super+s` Focus on the next workspace 1 - 10 (down).
+* `Super+w` Focus on the previous workspace 10 - 1 (up).
+
+...multiple monitors -- Alt+Super+__
+* `Alt+Super+left` Set focus to the left monitor.
+* `Alt+Super+right` Set focus to the right monitor.
+* `Alt+Super+down` Set focus to the downward monitor.
+* `Alt+Super+up` Set focus to the upward monitor.
+
+...to different workspaces and windows via switching applications (gnome key-binding)
+* `Super+tab` Switch between applications
+
+#### Moving a window
+... in the current workspace
+* `Super+left` Move the current window to the left.
+* `Super+right` Move the current window to the right.
+* `Super+up` Move the current window to upper workspace (top).
+* `Super+down` Move the current window to lower workspace (bottom).
+* `Shift+Super+1`, `Shift+Super+2`, ... `Shift+Super+0` Move window to specific workspace  
+
+...to multiple monitors
+* `Shift+Super+a` Move the current window to the monitor on the left.
+* `Shift+Super+d` Move the current window to the monitor on the right.
+* `Shift+Super+w` Move the current window to the upper workspace.
+* `Shift+Super+s` Move the current window to the lower workspace.
+
+... alternate combinations for multiple monitors (gnome keybindings)
+* `Shift+Super+left` Move the current window to the left monitor.
+* `Shift+Super+right` Move the current window to the right monitor.
+* `Shift+Super+down` Move the current window to the downward monitor.
+* `Shift+Super+up` Move the current window to the upward monitor. 
```

Comment:
We shouldn't list the gnome-shell shortcuts here since they aren't the same between distributions and it's seem that you didn't updated to the new list. You can check them here https://docs.google.com/spreadsheets/d/1v0vjnapj6Wlg2axwjLpljNs2jhxsjaJ8BzlqSC-JuHk/edit#gid=0

But be carefull they are not all implemented yet.

Reply:
Thanks! this spreadsheet is handy! I was looking through the XML to find the bindings. I'll check my findings against the spreadsheet. I didn't know that the distributions defined the kb-shortcuts, I would not have expected that.